frontend/src/assets/jss/colorPalette.js

```diff
@@ -6,3 +6,4 @@ export const textColor = "#FFFFFF";
 export const secondaryTextColor = "#D2D2D2";
 export const activeTextColor = "#009C19";
 export const backgroundColor = "#1C1C1C";
+export const signupButtonColor = "#009C19";
```

Comment:
This would also be useful for later refactoring, for example replacing hard-coded values from the headers.

Reply:
Two lines above we define the same color which we use in the header buttons.

Reply:
Right, I didn't notice. I better put `const signupButtonColor = activeTextColor` into the `SignupButton` component, right?